SQL-compiler/src/main/codegen/config.fmpp

```diff
@@ -0,0 +1,94 @@
+# Structure has to follow the fields in Calcite's default_config.fmpp.
```

Comment:
Does this extend the existing parser?
Or replace it?

Reply:
It extends the base parser. So DbspParserImpl can parse things like `select * from xyz`. It will not handle DDL for example.